### PR TITLE
Hotels: New design for roompicker button

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/hotels/src/singleHotel/roomPicker/IncrementDecrement.js [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+  currency,
+  increment,
+  selectableCount,
+}: Props) {
+  return (
+    <View style={styles.container}>
+      <View style={styles.row}>
+        <View style={styles.priceContainer}>
+          <Text style={styles.countText}>
+            {Platform.select({
```
Comment (RobinCsl) [inline app/hotels/src/singleHotel/roomPicker/IncrementDecrement.js]: Is it possible to make it clearer that only the prop textTransform is added on Android? Maybe creating the prop before the return statement and then just injecting or spreading a prop object?

File: app/hotels/src/singleHotel/roomPicker/SelectButton.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  +currency: string | null,
+  +increment: () => void,
+|};
+
+export default function SelectButton({ increment, price, currency }: Props) {
+  return (
+    <Touchable
+      style={[
+        styles.container,
+        styles.borderRadiusLeft,
```
Comment (RobinCsl) [inline app/hotels/src/singleHotel/roomPicker/SelectButton.js]: I cannot see this in the styles object. (Idem for `styles.borderRadiusRight`)

File: packages/shared/src/buttons/IncrementDecrementButtons.js [hunk 3]
```diff
@@ -45,78 +35,60 @@ const Button = ({
 };
 
 export default function IncrementDecrementButtons(props: {|
-  onIncrement: () => void,
-  onDecrement: () => void,
-  number: number,
-  min?: number,
-  max?: number,
+  +onIncrement: () => void,
+  +onDecrement: () => void,
```
Comment (RobinCsl) [inline packages/shared/src/buttons/IncrementDecrementButtons.js]: Idem here.

File: packages/shared/src/buttons/IncrementDecrementButtons.js [hunk 4]
```diff
@@ -45,78 +35,60 @@ const Button = ({
...
+        touchable={!disableDecrement}
+        onPress={props.onDecrement}
+      />
+      {props.showNumber && (
+        <Text style={props.numberStyle}>
+          <Translation passThrough={props.number} />
+        </Text>
+      )}
+      <Button
+        icon={<TextIcon code="&#xe122;" orbit={true} style={styles.icon} />}
```
Comment (RobinCsl) [inline packages/shared/src/buttons/IncrementDecrementButtons.js]: I am just starting to question adding this prop `orbit={true}` everywhere. Won't it be annoying once the migration to Orbit icons is complete to remove everything? Just thinking out loud.

File: app/hotels/src/singleHotel/roomPicker/IncrementDecrement.js (hunk 1 above)
Comment (tbergq) [reply app/hotels/src/singleHotel/roomPicker/IncrementDecrement.js]: Could you show with an example, I am not sure what you mean 😄

File: app/hotels/src/singleHotel/roomPicker/SelectButton.js (hunk 2 above)
Comment (tbergq) [reply app/hotels/src/singleHotel/roomPicker/SelectButton.js]: Yes, this is the result of copy/paste, I deleted them now

File: packages/shared/src/buttons/IncrementDecrementButtons.js (hunk 3 above)
Comment (tbergq) [reply packages/shared/src/buttons/IncrementDecrementButtons.js]: What?

File: packages/shared/src/buttons/IncrementDecrementButtons.js (hunk 4 above)
Comment (tbergq) [reply packages/shared/src/buttons/IncrementDecrementButtons.js]: Just search and replace orbit={true} -> ''

File: app/hotels/src/singleHotel/roomPicker/IncrementDecrement.js (hunk 1 above)
Comment (RobinCsl) [reply app/hotels/src/singleHotel/roomPicker/IncrementDecrement.js]: Of course.
So before `return`, have something like: 
```
const myAndroidProps = Platform.OS === 'ios' ? {} : {textTransform: 'uppercase'};
```

and then just write 
```
<Translation id="the.id" {...myAndroidProps} />
```

File: packages/shared/src/buttons/IncrementDecrementButtons.js (hunk 4 above)
Comment (tbergq) [reply packages/shared/src/buttons/IncrementDecrementButtons.js]: but yeah, maybe it is time to go the extra mile and just remove the old icons

File: packages/shared/src/buttons/IncrementDecrementButtons.js (hunk 3 above)
Comment (RobinCsl) [reply packages/shared/src/buttons/IncrementDecrementButtons.js]: Oh it looks like my other comment was not taken into account: I meant for you to maybe extract the Props type to make it more readable.

File: packages/shared/src/buttons/IncrementDecrementButtons.js (hunk 4 above)
Comment (RobinCsl) [reply packages/shared/src/buttons/IncrementDecrementButtons.js]: There's only 92 files to go through... 🍰

**Comment by tbergq** [comment]
@RobinCsl 🚀

**Pull request merged**